update.sh

```diff
@@ -53,9 +53,9 @@ if [ ! -x "$(which systemctl)" ]; then
     exit 1
 fi
 
-# stop the systemd service now, because after the update
+# stop the containers now, because after the update
 # the code might be out of sync with the current running services
-systemctl stop "${SERVICE_NAME}"
+./stop.sh
```

Comment:
Or we could remove the systemd support altogether, this would reduce the requirement of testing on two different types of systemd, just like the idea behind using Docker in the first place.
Removing the systemd support isn't that complicated, display a warning message to use the new systemd if the user use the original systemd service file and make the `start.sh` script inefficient when started by systemd.

Reply:
> we could remove the systemd

I don't think [a file](https://github.com/searx/searx-docker/blob/master/searx-docker.service.template), few command lines to explain how to install this project using systemd won't hurt.

But I do agree to put on the side the systemd stuff.